domains/libraries/EnergyOrigin.Setup/EnergyOrigin.Setup.Tests/Migrations/DbMigratorTest.cs

```diff
@@ -0,0 +1,105 @@
+using EnergyOrigin.Setup.Migrations;
+using EnergyTrackAndTrace.Testing.Testcontainers;
+using Microsoft.Extensions.Logging.Abstractions;
+using Npgsql;
+using Xunit;
+
+namespace EnergyOrigin.Setup.Tests.Migrations;
+
+public class DbMigratorTest : IClassFixture<PostgresContainer>
+{
+    private readonly DatabaseInfo _databaseInfo;
+    private readonly DbMigrator _dbMigrator;
+
+    public DbMigratorTest(PostgresContainer postgresContainer)
+    {
+        _databaseInfo = postgresContainer.CreateNewDatabase().GetAwaiter().GetResult();
+        _dbMigrator = new DbMigrator(_databaseInfo.ConnectionString, typeof(DbMigratorTest).Assembly, NullLogger<DbMigrator>.Instance);
+    }
+
+    [Fact]
+    public async Task GivenNewDatabase_WhenApplyingMigrations_AllMigrationsAreApplied()
+    {
+        await MigrateSuppressExceptionsAsync();
+
+        var scripts = await GetAppliedMigrations();
+        Assert.True(scripts.Count == 2);
+        Assert.Equal("20250116-0001-NoOp1.sql", scripts.First());
+        Assert.Equal("20250116-0010-NoOp2.sql", scripts.Last());
+    }
+
+    [Fact]
+    public async Task GivenNewDatabase_WhenApplyingFailingMigration_LaterMigrationsAreNotApplied()
+    {
+        await MigrateSuppressExceptionsAsync();
+
+        var scripts = await GetAppliedMigrations();
+        Assert.True(scripts.Count == 2);
```

Comment:
i think these assertions are hard to read. Why 2? and why not 001 and 0010 is it the first and the last?